src/actors/snapshots/symbolicator__actors__symbolication__tests__wasm_payload.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: src/actors/symbolication.rs
+expression: response
+---
+status: completed
+stacktraces:
+  - frames:
+      - status: symbolicated
+        original_index: 0
+        addr_mode: "rel:0"
+        instruction_addr: 0x8c
+        lang: rust
+        symbol: internal_func
+        sym_addr: 0x8b
+        function: internal_func
+        filename: src/lib.rs
+        abs_path: /Users/mitsuhiko/Development/wasm-example/simple/src/lib.rs
+        lineno: 19
+modules:
```

Comment:
I'm in two minds on whether this should include the dif candidates section or not

Reply:
Good question. It would be nice to have dedicated tests in this actor for candidates that also asserts the different states for unwind & debug.